Azure/azure-cli
# [AppService] Fix #13261 az webapp list-runtimes use static list until new Available Stacks API is available

**Description**

Currently `az webapp list-runtimes` returns a list of runtimes provided by the Available Stacks API. However, the Available Stacks API is updated with Antares deployments which are infrequent and don't line up with stacks EOL schedule.

We want to return a list of static runtimes until the Available Stacks API is updated.

**Testing Guide**  

`az webapp list-runtimes` (--linux) should return the list shown in https://github.com/Azure/azure-cli/issues/13261

---

This checklist is used to make sure that common guidelines for a pull request are followed.
